README.md

```diff
@@ -1 +1,34 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+## Portfolio-setup-and-mobile-version-skeleton
+
+## Built With
+
+- HTML
+- CSS
+- Lighthouse
+- Webhint
+- Stylelint
+
+## Getting Started
+
+#### cloning project
+git clone git@github.com:collinsmezie/Portfolio-setup-and-mobile-version-skeleton.git
+
+## Getting packages and debuging with Stylelint npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x 
+
+
+##### For validation detection using Stylelint Run npx stylelint "**/*.{css,scss}" 
+
+##### from parent source directory 
+
+## Getting packages and debuging with Webhint npm init -y npminstall --save-dev hint@6.x 
+
+##### For validation detection using Webhint Run npx hint . 
```

Comment:
- [ ] Please edit the readme paragraphs so that it was clear to understand the project for anyone who opens you repository. try to use headings `###` and after that just put text expanding the title idea. in case of getting started section I suggest :

```
## Getting Started

To get a local copy up and running follow these simple example steps.

- go to the directory that you want to clone the repository in.
- right click and choose Git Bash here.
- type "git clone https://github.com/collinsmezie/Portfolio-setup-and-mobile-version-skeleton.git".
```